platform/darwin/src/MGLBackgroundStyleLayer.mm

```diff
@@ -10,6 +10,13 @@
 
 #include <mbgl/style/layers/background_layer.hpp>
 
+namespace mbgl {
```

Comment:
Can we get rid of this block if there’s are no symbols to define in this translation unit?